src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter(
+      person => person.sex === 'm' && Math.ceil(person.died / 100) === century
+    )
+    : people.filter(person => person.sex === 'm');
```

Comment:
делай проверку на century внутри самого filter, тогда не надо будет его дублировать
```suggestion
  const men = people.filter(
      person => century 
        ? person.sex === 'm' && Math.ceil(person.died / 100) === century
        : person.sex === 'm'
  );
    
```